### PR TITLE
8.0 mgmtsystem review

### PR DESCRIPTION
**Pull request by max3903**
Depends on 
- [x] #30 
- [x] #25

### CODE REVIEW

**Comment by pedrobaeza** [comment]
migrations/7.0.1.0/ folder is not needed anymore.

Please change icon path and check Travis.

**Comment by llacroix** [comment]
Depends on 
- [ ] #30 
- [ ] #25

**Comment by dreispt** [comment]
Waiting on dependencies.

File: mgmtsystem_review/models/mgmtsystem_review.py [hunk 1]
```diff
@@ -19,7 +19,7 @@
 #
 ##############################################################################
 
-from osv import fields, orm
+from openerp.osv import fields, orm
```
Comment (dreispt) [inline mgmtsystem_review/models/mgmtsystem_review.py]: Is osv still needed?

Comment (llacroix) [inline mgmtsystem_review/models/mgmtsystem_review.py]: It can probably get removed

**Comment by dreispt** [comment]
:+1:

**Comment by max3903** [comment]
:+1:

**Pull request merged**